packages/volto/src/actions/content/content.js

```diff
@@ -25,9 +25,10 @@ import config from '@plone/volto/registry';
  * @param {number} number of files uploaded
  * @returns {Object} update uploaded files action
  */
-export function updateUploadedFiles(uploadedFiles) {
+export function updateUploadedFiles(uploadedFiles, subrequest) {
   return {
     type: UPDATE_UPLOADED_FILES,
+    subrequest,
```

Comment:
Here, why?

Reply:
I thought I'll pass 'batch-upload' as a subrequest to `updateUploadedFiles` action, then the reducer would use it to update the `subrequests['batch-upload']` to something like this 
```
subrequests: {
'batch-upload': {
   data: null,
   error: null,
   loading: true,
   loaded: false,
   uploadedFiles: 1
}
```